portfolio/src/main/webapp/script.js

```diff
@@ -49,4 +49,17 @@ function openPage(pageName, elmnt, color) {
 }
 
 // Get the element with id="defaultOpen" and click on it
-document.getElementById("defaultOpen").click();
+async function fetchServlet(){
+    const commentList = document.getElementById("comment");
```

Comment:
does your page have an element with this ID? If not, the browser will be unable to find the element and your function with error